riva/proto/riva_asr.proto

```diff
@@ -170,6 +175,23 @@ message StreamingRecognitionConfig {
     bool interim_results = 2;
 }
 
+// Config to enable speaker diarization.
+message SpeakerDiarizationConfig {
+  // If 'true', enables speaker detection for each recognized word in
+  // the top alternative of the recognition result using a speaker_tag provided
+  // in the WordInfo.
+  bool enable_speaker_diarization = 1;
+
+  // Minimum number of speakers in the conversation. This range gives you more
+  // flexibility by allowing the system to automatically determine the correct
+  // number of speakers. If not set, the default value is 2.
+  int32 min_speaker_count = 2;
+
+  // Maximum number of speakers in the conversation. This range gives you more
+  // flexibility by allowing the system to automatically determine the correct
+  // number of speakers. If not set, the default value is 6.
+  int32 max_speaker_count = 3;
```

Comment:
What do `max_speaker_count` and `min_speaker_count` map to in the nemo code? `max_speaker_count` is clear.  What does `min_speaker_count` do?